content/docs/06-integrations/00-prometheus-operator.md

```diff
@@ -36,6 +36,165 @@ We recommend you use version v44.3.x or greater moving forward for a simplified
+
+<br />
+
+```yaml
+charts:
+  kube-prometheus-stack:
+    alertmanager:
+      config:
+        receivers:
+          - name: email-alert
+            email_configs:
+              - to: <reciever>@<domainname>.com
+                send_resolved: true
+                from: <sender>@<domainname>.com
+                smarthost: smtp.<domainname>.com:587
+                auth_username: <sender>@<domainname>.com
+                auth_identity: <sender>@<domainname>.com
+                auth_password: <sender_passwd>
+```
+
+Refer to the [Prometheus Alertmanager Configuration](https://prometheus.io/docs/alerting/latest/configuration/) documentation to learn more about Alertmanager.
+
+<br />
+
+#### Grafana Ingress
+
+You can enable an ingress endpoint for Grafana that will deploy an NGINX ingress controller. This feature can be used to enable HTTPS and require authentication for all Prometheus API requests. 
+
+If you do not enable the ingress option, then by default a service with a load balancer will be created that exposes port 80. 
+
+
+Toggle the **Enable** button to enable the use of Ingress.
+
+<br />
+
+
+#### Thanos SideCar
+
+[Thanos](https://prometheus-operator.dev/docs/operator/thanos/) is an open-source system for running large-scale, distributed, and highly available Prometheus setups. Thanos allows Prometheus to store data for extended periods in object storage, such as Amazon S3 or Google Cloud Storage, instead of a local disk. This enables Prometheus to scale horizontally without the risk of using up local storage space.
+
+
+Toggle the **Enable** button to enable the use of Thanos.
+
+<br />
+
+#### Object Store
+
+Select the Thanos object storage type you will use. Review the `thanos.objstoreConfig` parameters to configure the use of object storage with Thanos. Refer to the [Thanos Object Storage](https://github.com/thanos-io/thanos/blob/main/docs/storage.md) documentation to learn more about how to configure each object storage.
+
+<br />
+
+```yaml
+charts:
+  kube-prometheus-stack:
+    prometheus:
+      prometheusSpec:
+        thanos:
+          objstoreConfig:
+```
+
+#### Thanos Ruler Object Store
+
+By default, Thanos Ruler event data is saved in object storage specified for Thanos, but you can specify a different object storage for event data.  Refer to the [Thanos Ruler](https://prometheus-operator.dev/docs/operator/thanos/?#thanos-ruler) resource to learn more.
+
+
+<br />
+
+#### Remote Monitoring
+
+You can configure the Prometheus server to accept metrics from Prometheus agents and become a centralized aggregation point for all Kubernetes metrics. Enabling this feature will expose port 9090 of the *prometheus-operator-prometheus* service. Use the generated service URL to provide other Kubernetes clusters with the [Prometheus Agent](/integrations/prometheus-agent) installed so that cluster metrics can be forwarded to the Prometheus server. 

```

Comment:
```suggestion
You can configure the Prometheus server to accept metrics from Prometheus agents and become a centralized aggregation point for all Kubernetes metrics. Enabling this feature will expose port 9090 of the *prometheus-operator-prometheus* service. Use the generated service URL to provide other Kubernetes clusters with the installed [Prometheus Agent](/integrations/prometheus-agent) so that cluster metrics can be forwarded to the Prometheus server. 

```